### ISSUE TITLE
Welcome screen needs a close button

### ISSUE THREAD

**Issue by Banbury** [wonderunit/storyboarder#483]
I can't seem to find a close button for the welcome screen. This leaves the user wondering, how to exit the program.

**Comment by setpixel** [comment]
Can you post a screenshot? It should be in the upper left.

Are you on windows?

**Comment by Banbury** [comment]
Now that you pointed it out, I've found it. But it's tiny and white and almost invisible. I'd suggest making it black or dark grey and about twice as big.
And yes I'm on Windows 7.

**Comment by setpixel** [comment]
Yeah - we are going to style it differently for windows. (right side, more like a button). on mac it's easy to see.
